go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/orbs-network/gamma-cli
+
+go 1.13
+
+require (
+	github.com/orbs-network/lean-helix-go v0.2.3
```

Comment:
Why lean helix?

Reply:
it is need for block proof in crypto/reciept_proof.go